.github/workflows/auto_update_charm_libs.yaml

```diff
@@ -24,14 +24,27 @@ jobs:
           charmcraft fetch-lib
         env:
           CHARMCRAFT_AUTH: ${{ secrets.CHARMHUB_TOKEN }}
+      - name: Import GPG key
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v5
```

Comment:
Could we use something like this to avoid having a dependency get access to one of our GPG keys? https://gist.github.com/vansergen/88eb7e71fea2e3bdaf6aa3e752371eb7